src/curategpt/app/app_alz.py

```diff
@@ -0,0 +1,303 @@
+"""Streamlit app for CurateGPT."""
+
+import json
+import logging
+from typing import List, Union
+
+import numpy as np
+import streamlit as st
+import yaml
+from scipy.spatial import distance_matrix
+
+from curategpt import BasicExtractor
+from curategpt.agents import MappingAgent
+from curategpt.agents.bootstrap_agent import BootstrapAgent, KnowledgeBaseSpecification
+from curategpt.agents.chat_agent import ChatAgent, ChatResponse
+from curategpt.agents.dase_agent import DatabaseAugmentedStructuredExtraction
+from curategpt.agents.dragon_agent import DragonAgent
+from curategpt.agents.evidence_agent import EvidenceAgent
+from curategpt.app.components import (
+    DimensionalityReductionOptions,
+    limit_slider_component,
+    vectors_to_fig,
+)
+from curategpt.app.helper import get_applicable_examples, get_case_collection
+from curategpt.app.state import get_state
+from curategpt.extract import OpenAIExtractor, RecursiveExtractor
+from curategpt.wrappers import BaseWrapper
+from curategpt.wrappers.literature import WikipediaWrapper
+from curategpt.wrappers.literature.pubmed_wrapper import PubmedWrapper
+
+PUBMED = "PubMed (via API)"
+WIKIPEDIA = "Wikipedia (via API)"
+# Removed JGI and ESS-Dive
+# JGI = "JGI (via API)"
+# ESSDIVE = "ESS-DeepDive (via API)"
+
+CHAT = "Chat"
+SEARCH = "Search"
+
+# Removed other operations
+# EXTRACT = "Extract"
+# CLUSTER_SEARCH = "Cluster Search"
+# MATCH = "Match"
+# BOOTSTRAP = "Bootstrap"
+# CURATE = "Curate"
+# ADD_TO_CART = "Add to Cart"
+# CITESEEK = "CiteSeek"
+# CART = "Cart"
+# HELP = "Help"
+# EXAMPLES = "Examples"
+# ABOUT = "About"
+
+NO_BACKGROUND_SELECTED = "No background collection"
+
+MODELS = [
```

Comment:
For demo purposes we could just remove model selection but display which model is in use

Reply:
+1 done